mpldxf/backend_dxf.py

```diff
@@ -479,6 +476,21 @@ def new_gc(self):
     def points_to_pixels(self, points):
         return points / 72.0 * self.dpi
 
+    def draw_quad_mesh(
```

Comment:
When I tried to figure out where the failure happend, I tried some different approaches and figured out that we didnt handle quad mesh, so I set up some of the entities we didnt handle to see what wass passed to them.
Thought we just could keep them to be aware of them and implement them if necessary at a later stage.
But I will just remove it for now, since not in use.